cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -76,6 +78,55 @@ def local_ansible_catalog_item(appliance, ansible_repository):
     cat_item.delete_if_exists()
 
 
+@pytest.fixture(scope="function")
+def dialog_with_catalog_item(appliance, ansible_repository, ansible_catalog):
+    service_dialog = appliance.collections.service_dialogs
+    dialog = fauxfactory.gen_alphanumeric(12, start="dialog_")
+    ele_name = "service_name"
```

Comment:
please use `fauxfactory` to create dynamic names

Reply:
Here `fauxfactory` is not required, as `"service_name"` will be used for Automate domain, which will process dialog element field with the given service name. It will assign entered service name to the service.